src/mumble/MainWindow.cpp

```diff
@@ -2335,12 +2339,33 @@ void MainWindow::on_qaChannelListen_triggered() {
 	}
 }
 
-void MainWindow::on_qaChannelFilter_triggered() {
+void MainWindow::on_qaChannelHide_triggered() {
+	Channel *c = getContextMenuChannel();
+
+	if (c) {
+		UserModel *um = static_cast< UserModel * >(qtvUsers->model());
+		if (qaChannelHide->isChecked()) {
+			c->setFilterMode(Channel::FILTER_MODE_HIDE);
+			qaChannelPin->setChecked(false);
```

Comment:
Not sure if this is actually needed as the context menu will have to be re-opened anyway at which point the check-state of the checkboxes should be re-evaluated anyway :thinking: